docs/testing/regression-checklist/mapping.json

```diff
@@ -30,7 +30,7 @@
 		"testFile": [ "extensions/blocks/checklist.md" ]
 	},
 	{
-		"regex": "^projects\/plugins\/jetpack\/extensions\/(blocks\/(subscriptions|premium-content)\/|shared\/memberships\.js)",
+		"regex": "^projects/plugins/jetpack/extensions/(blocks/(subscriptions|premium-content)/|shared/memberships.js)",
```

Comment:
The version in trunk is different.
https://github.com/Automattic/jetpack/blob/21c9c9d586c08e25af92f5de3d8122234f4fb4ff/docs/testing/regression-checklist/mapping.json#L33

Reply:
@pyronaur which version is correct? The other regexes in the file aren't escaped.

Reply:
We probably need to revert this

Reply:
Ah, I see what you mean, yeah - the others aren't escaped, so i think this'll be fine?

Reply:
Prettier formatting did this. Closing.